schema/mysql/schema.sql

```diff
@@ -1012,11 +1012,15 @@ CREATE TABLE kubernetes_instance (
 CREATE TABLE config (
   cluster_uuid binary(16) NOT NULL,
   `key` enum(
-    'notifications.url',
```

Comment:
The diff looks a little strange. `notifications.url` must not be moved and `notifications.source_id` must not be added. Please leave the original values unchanged and only add the new keys.